libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -330,7 +335,9 @@ public void onSaveInstanceState(Bundle outState) {
         outState.putBoolean(KEY_IS_SOCIAL, mIsSocialLogin);
         outState.putBoolean(KEY_IS_DISPLAYING_EMAIL_HINTS, mIsDisplayingEmailHints);
         outState.putBoolean(KEY_HAS_DISMISSED_EMAIL_HINTS, mHasDismissedEmailHints);
-        outState.putBoolean(VALIDITY_EMAIL, mIsValidEmail);
+        if (mIsValidEmail != null) {
```

Comment:
You don't have to check if this is null, you can simply just put it similar to the others above 😄

Reply:
@jd-alexander Unfortunately that's not entirely correct, because it looks like `mIsValidEmail` is a `Boolean` and not a `boolean`. I think that created a bit of a confusion here.

@FernandaCG Could you update the `mIsValidEmail`'s type to `boolean` and remove the `null` check here? You should be able to follow the example of `mIsSocialLogin` for the other parts as it should work similarly.

Let me know if either of you have any questions.

Reply:
Thanks @oguzkocer I just realized my mistake 🤦‍♂ Yes, since `Boolean` is the wrapper type the check has to be made. I was looking at the values above and got confused.